README.md

```diff
@@ -38,6 +38,12 @@ $ go get -u github.com/davecgh/go-spew/spew
 
 ## Quick Start
 
+Add this import line to the file you're working in:
+
+```Go
+import spew "github.com/davecgh/go-spew/spew"
```

Comment:
Thanks for the PR.

One note:
The initial spew isn't needed.  It will already be named that.

```Go
import "github.com/davecgh/go-spew/spew"
```